src/tailwindcss-stubs/resources/views/auth/passwords/reset.blade.php

```diff
@@ -20,7 +20,7 @@
                                 {{ __('E-Mail Address') }}:
                             </label>
 
-                            <input id="email" type="email" class="form-input w-full @error('email') border-red-500 @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus>
+                            <input id="email" type="email" class="form-input w-full @error('email') border-red-500 @enderror" name="email" value="{{ old('email', request()->email) }}" required autocomplete="email" autofocus>
```

Comment:
Probably better if we use the same structure as what's provided in [`laravel/ui`](https://github.com/laravel/ui/blob/2.x/src/Auth/bootstrap-stubs/auth/passwords/reset.stub#L20)